tmt/steps/provision/local.py

```diff
@@ -28,6 +29,17 @@ class GuestLocal(tmt.Guest):
     localhost = True
     parent: Optional[tmt.steps.Step]
 
+    @property
+    def scripts_path(self) -> Path:
+        """
+        Absolute path to tmt scripts directory
+        """
+
+        # TODO: For now we ignore the 'TMT_SCRIPTS_DIR' variable.
+        # To be fixed in https://github.com/teemtee/tmt/issues/4081
+        parent = cast(Provision, self.parent)
```

Comment:
It's generally safer to use `assert isinstance()` instead of `cast()`. Both provide hint for a type checker, but `isinstance()` will fail in runtime if the assumption is incorrect. With `cast()`, you are basically saying `self.parent` is `Provision`, and it can never ever change, it's like a law of physics. If you are wrong, nothing will let you know you were wrong. On the other hand, if you were wrong about this "it's always `Provision` statement", `isinstance()` will raise an error, and it will let you know what the Universe thinks about your absolute statements.